src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -43,7 +55,9 @@ public void periodic(){
     // TODO 2.2.5: Update odometry
 
     // TODO 1.2.2: Call tankDrive()
-
+        tankDrive(
+        Robot.driver.getLeftTranslation(),
+        Robot.driver.getRightTranslation());
```

Comment:
It might be good to indent this in the future just for readability. Example below.

```java
 tankDrive(
        Robot.driver.getLeftTranslation(),
        Robot.driver.getRightTranslation()
);
```